package.json

```diff
@@ -11,8 +11,8 @@
     "purge": "TS_NODE_FILES=true ts-node --files scripts/purge",
     "start": "ts-node --files src/app.ts",
     "start:prod": "./node_modules/.bin/pm2 start ecosystem.config.js --update-env",
-    "test": "unset IMDB_API_KEY && OS_API_USERAGENT=OS_API_MOCK_KEY jest test/e2e/media-video.spec.ts --coverage --forceExit --detectOpenHandles && npm run test:mocks",
-    "test:mocks": "unset IMDB_API_KEY && OS_API_USERAGENT=OS_API_MOCK_KEY jest test/**/*.spec.ts  --testPathIgnorePatterns test/e2e/media-video.spec.ts --coverage --forceExit --detectOpenHandles",
+    "test": "unset IMDB_API_KEY && unset NODE_ENV && OS_API_USERAGENT=OS_API_MOCK_KEY jest test/e2e/media-video.spec.ts --coverage --forceExit --detectOpenHandles && npm run test:mocks",
```

Comment:
How come this is needed? Jest injects NODE_ENV with `test`

Reply:
@js-kyle Jest only injects that value if `NODE_ENV` is not set already https://jestjs.io/docs/environment-variables#node_env